src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackVersion.java

```diff
@@ -1,6 +1,25 @@
 package uk.gov.hmcts.reform.civil.callback;
 
+import java.util.Arrays;
+import java.util.List;
+
 public enum CallbackVersion {
     V_1,
-    V_2
+    V_2;
+
+    /**
+     * Checks whether a version number is at least this version's number.
+     *
+     * @param anotherVersion a version number
+     * @return true if anotherVersion is not null and anotherVersion is this or above
+     */
+    public boolean isEqualOrGreater(CallbackVersion anotherVersion) {
```

Comment:
@dtortolaV1 It would be nice to have unit tests for this method

Reply:
Added